codeworld-api/src/CodeWorld.hs

```diff
@@ -121,7 +121,8 @@ module CodeWorld (
     pattern PointerMovement,
 
     -- * Debugging
-    trace
+    trace,
+    addMessage
```

Comment:
Can you add a CodeWorld.Internal package, and only export from that?  I don't really want this to appear as part of the public CodeWorld API.